packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.test.tsx

```diff
@@ -495,4 +495,18 @@ describe('ComboBox', () => {
 
     expect(updatedText).toEqual('');
   });
+
+  it('invokes optional onItemClick callback on option select', () => {
+    const onItemClickMock = jest.fn();
+    const wrapper = mount(<ComboBox options={DEFAULT_OPTIONS} onItemClick={onItemClickMock} />);
+    wrapper.find('input').simulate('keydown', { which: KeyCodes.enter });
+    const buttons = document.querySelectorAll('.ms-ComboBox-option');
+
+    // ComboBox components are not being cleaned up per test.
+    // Therefore, select the last option button which belongs
+    // to this ComboBox's unit test and invoke 'click' on it.
+    (buttons[buttons.length - 1] as HTMLButtonElement).click();

```

Comment:
Unrelated to this change, it looks like `ComboBox` are not cleaned-up per unit test currently. I realized this because the `length` of `.ms-ComboBox-option` in the DOM is `28` here. It should be `4` as there are `4` options.

Reply:
Fixed in https://github.com/OfficeDev/office-ui-fabric-react/pull/6927/commits/26a55577a41ced771563941b7e318e21d7ed5e08.